resources/guides/uwinsite_guide.md

```diff
@@ -0,0 +1,236 @@
+---
+id: uwinsite_guide
+title: UWinsite Guide
+sidebar_label: UWinsite Guide
+slug: /guides/uwinsite_guide
+---
+
```

Comment:
Please include a `_Last updated on ..._` . You can read the specific format that we want in [REQUIREMENTS.md](https://github.com/UWindsorCSS/wiki/blob/master/REQUIREMENTS.md)